Cargo.toml

```diff
@@ -101,38 +73,3 @@ opt-level = 3 # BLS library is too slow to use in debug
-memory_stats = ["neard/memory_stats"]
-c_memory_stats = ["neard/c_memory_stats"]
-expensive_tests = ["neard/expensive_tests"]
-regression_tests = []
-old_tests = []
-adversarial = ["neard/adversarial", "near-jsonrpc/adversarial", "near-store/adversarial"]
-no_cache = ["neard/no_cache"]
-metric_recorder = ["neard/metric_recorder"]
-delay_detector = ["neard/delay_detector"]
-rosetta_rpc = ["neard/rosetta_rpc"]
-nightly_protocol = ["near-primitives/nightly_protocol", "near-jsonrpc/nightly_protocol", "testlib/nightly_protocol", "neard/nightly_protocol"]
-nightly_protocol_features = ["nightly_protocol", "neard/nightly_protocol_features", "protocol_feature_evm", "protocol_feature_block_header_v3", "protocol_feature_alt_bn128", "protocol_feature_add_account_versions", "protocol_feature_tx_size_limit", "protocol_feature_allow_create_account_on_delete", "testlib/nightly_protocol_features", "protocol_feature_fix_storage_usage", "protocol_feature_restore_receipts_after_fix", "protocol_feature_cap_max_gas_price"]
-protocol_feature_evm = ["neard/protocol_feature_evm", "testlib/protocol_feature_evm", "runtime-params-estimator/protocol_feature_evm"]
-protocol_feature_alt_bn128 = ["neard/protocol_feature_alt_bn128", "testlib/protocol_feature_alt_bn128", "runtime-params-estimator/protocol_feature_alt_bn128"]
-protocol_feature_block_header_v3 = ["near-primitives/protocol_feature_block_header_v3", "near-chain/protocol_feature_block_header_v3", "neard/protocol_feature_block_header_v3"]
-protocol_feature_tx_size_limit = ["near-primitives/protocol_feature_tx_size_limit", "node-runtime/protocol_feature_tx_size_limit", "neard/protocol_feature_tx_size_limit"]
-protocol_feature_allow_create_account_on_delete = ["testlib/protocol_feature_allow_create_account_on_delete", "near-primitives/protocol_feature_allow_create_account_on_delete", "node-runtime/protocol_feature_allow_create_account_on_delete", "neard/protocol_feature_allow_create_account_on_delete"]
-protocol_feature_add_account_versions = ["near-primitives/protocol_feature_add_account_versions"]
-protocol_feature_fix_storage_usage = ["near-primitives/protocol_feature_fix_storage_usage", "neard/protocol_feature_fix_storage_usage", "node-runtime/protocol_feature_fix_storage_usage"]
-protocol_feature_restore_receipts_after_fix = ["near-primitives/protocol_feature_restore_receipts_after_fix", "near-chain/protocol_feature_restore_receipts_after_fix", "neard/protocol_feature_restore_receipts_after_fix", "node-runtime/protocol_feature_restore_receipts_after_fix", "restored-receipts-verifier/protocol_feature_restore_receipts_after_fix"]
-protocol_feature_cap_max_gas_price = ["near-primitives/protocol_feature_cap_max_gas_price", "neard/protocol_feature_cap_max_gas_price", "near-chain/protocol_feature_cap_max_gas_price"]
-
-# enable this to build neard with wasmer 1.0 runner
-# now if none of wasmer0_default, wasmer1_default or wasmtime_default is enabled, wasmer0 would be default
```

Comment:
Those features are needed, why they are removed?

Reply:
Good catch, I didn't notice that! I think it'd be good indeed to move this over to `neard/Cargo.toml` instead of just deleting. 

That being said, I wouldn't block on that, as the following now works without feature forwarding:

```
cargo build -p neard --features node-runtime/wasmer1_default
```

Reply:
@olonho Great catch!

It seems it should be in both `nearcore` (defined as `node-runtime/wasmer1_default`) and in `neard` (defined as `nearcore/wasmer1_default`).